astropy/io/fits/connect.py

```diff
@@ -92,6 +94,10 @@ def read_table_fits(input, hdu=None):
         - :class:`~astropy.io.fits.hdu.hdulist.HDUList`
     hdu : int or str, optional
         The HDU to read the table from.
+    astropy_native : bool
+        The option to read in FITS columns as mixins if possible. By default
+        this option is set to False, to read raw fits data as non-mixin Table
```

Comment:
It is worth breaking this into two parts, with a longer narrative description above and then a shorter version here.  Start with the longer description:
```
If the ``astropy_native`` argument is ``True`` then input FITS columns which are representations
of an astropy core object will be converted to that class and stored in the table as a
"mixin column".  Currently this is limited to FITS columns which adhere to the 
FITS Time standard, in which case they will be converted to a `~astropy.time.Time`
column in the output table.
```
(In the above make sure the line length is shorter than I have done).

Then the short part is just:
```
astropy_native : bool, optional
    Read in FITS columns as native astropy objects where possible instead of standard Table
    Column objects.  Default is False.
```